cms/static/js/index.js

```diff
@@ -62,6 +62,7 @@ define(['domReady', 'jquery', 'underscore', 'js/utils/cancel_on_escape', 'js/vie
             analytics.track('Created a Course', course_info);
             CreateCourseUtils.create(course_info, function(errorMessage) {
                 $('.create-course .wrap-error').addClass('is-shown');
+                // xss-lint: disable=javascript-jquery-html
                 $('#course_creation_error').html('<p>' + errorMessage + '</p>');
```

Comment:
Do you know that errorMessage is safe and doesn't contain html that should be escaped?  Please remove this disable if you aren't sure.